src/main/CommandHandler.cpp

```diff
@@ -1174,6 +1174,68 @@ CommandHandler::generateLoad(std::string const& params, std::string& retStr)
         cfg.dexTxPercent =
             parseOptionalParamOrDefault<uint32_t>(map, "dextxpercent", 0);
 
+        // Only for SOROBAN_INVOKE mode, resource consumption bounds
+        cfg.nDataEntriesLow =
+            parseOptionalParamOrDefault<uint32_t>(map, "dataentrieslow", 0);
+        cfg.nDataEntriesHigh =
+            parseOptionalParamOrDefault<uint32_t>(map, "dataentrieshigh", 0);
+        cfg.kiloBytesPerDataEntryLow =
+            parseOptionalParamOrDefault<uint32_t>(map, "kilobyteslow", 0);
+        cfg.kiloBytesPerDataEntryHigh =
+            parseOptionalParamOrDefault<uint32_t>(map, "kilobyteshigh", 0);
+        cfg.txSizeBytesLow =
+            parseOptionalParamOrDefault<uint32_t>(map, "txsizelow", 0);
+        cfg.txSizeBytesHigh =
+            parseOptionalParamOrDefault<uint32_t>(map, "txsizehigh", 0);
+        cfg.instructionsLow =
+            parseOptionalParamOrDefault<uint64_t>(map, "cpulow", 0);
+        cfg.instructionsHigh =
+            parseOptionalParamOrDefault<uint64_t>(map, "cpuhigh", 0);
+
+        // SOROBAN_CREATE_UPGRADE parameters
+        cfg.maxContractSizeBytes =
+            parseOptionalParamOrDefault<uint32_t>(map, "mxcntrctsz", 0);
+        cfg.maxContractDataKeySizeBytes =
+            parseOptionalParamOrDefault<uint32_t>(map, "mxcntrctkeysz", 0);
+        cfg.maxContractDataEntrySizeBytes =
+            parseOptionalParamOrDefault<uint32_t>(map, "mxcntrctdatasz", 0);
+        cfg.ledgerMaxInstructions =
+            parseOptionalParamOrDefault<uint64_t>(map, "ldgrmxinstrc", 0);
+        cfg.txMaxInstructions =
+            parseOptionalParamOrDefault<uint64_t>(map, "txmxinstrc", 0);
+        cfg.txMemoryLimit =
+            parseOptionalParamOrDefault<uint64_t>(map, "txmemlim", 0);
+        cfg.ledgerMaxReadLedgerEntries =
+            parseOptionalParamOrDefault<uint32_t>(map, "ldgrmxrdntry", 0);
+        cfg.ledgerMaxReadBytes =
+            parseOptionalParamOrDefault<uint32_t>(map, "ldgrmxrdbyt", 0);
+        cfg.ledgerMaxWriteLedgerEntries =
+            parseOptionalParamOrDefault<uint32_t>(map, "ldgrmxwrntry", 0);
```

Comment:
nit:  This is really hard to read... Is it really necessary to get rid of most of the vowels here? Why not just use underscore and proper long names?

Reply:
Unfortunately this is necessary. For supercluster runs, we set all loadgen parameters via a single HTTP string. With the addition of these config parameters, we're running up against the HTTP string limits. If these names are longer, we'd have to break network upgrades into several small upgrades, which slows down runs and increases the chance of failure, since loadgen in supercluster is a little flaky to begin with.

Reply:
Well, maybe it's time to switch to POST request :) That's fine for now then.